tests/check-locales.test.js

```diff
@@ -0,0 +1,209 @@
+const mockFs = {
+  readdirSync: jest.fn(),
+  statSync: jest.fn(),
+  readFileSync: jest.fn()
+};
+
+jest.mock('fs', () => mockFs);
+
+jest.mock('../scripts/utils/logger', () => ({
+  logger: {
+    error: jest.fn(),
+    warn: jest.fn(),
+    info: jest.fn()
+  }
+}));
```

Comment:
Add these mocks after imports.

Reply:
mockFs needs to be hoisted for it to be available to file system mock